docs/tutorials/services/navigate-with-rover-base.md

```diff
@@ -0,0 +1,646 @@
+Viam's motion planner will plan routes that avoid those obstacles and attempt to keep the robot at your specified velocity.
+
+To try it out yourself, you need a mobile base and a movement sensor that can track the robot's GPS coordinates and angular and linear velocity.
+Follow this tutorial to get started using Viam's Navigation service to help your wheeled base navigate across space with our recommended stack.
+
+{{<video webm_src="/tutorials/navigate-with-rover-base/agilex-limo-roving.webm" mp4_src="/tutorials/navigate-with-rover-base/agilex-limo-roving.mp4" alt="The agilex rover navigating outside" poster="/tutorials/navigate-with-rover-base/agilex-roving.png">}}
+
+## Requirements
+
+1. **A wheeled base with encoded motors**
+
+   We used [a LEO rover](https://www.leorover.tech/shop?gclid=CjwKCAjw38SoBhB6EiwA8EQVLiDwUFEYgLxaRd1-TiTyLfifIAHs9iD6YnvdW6M-3rXruHOrzfTL2RoCD1AQAvD_BwE), configured as a [`wheeled` base](/components/base/wheeled/), but you can use whatever model of rover base you have on hand:
+
+   {{<imgproc src="/tutorials/navigate-with-rover-base/leo-in-office.png" resize="500x" declaredimensions=true alt="Leo rover that is navigating using the navigation service in a robotics lab.">}}
+
+2. **A movement sensor with GPS position and angular and linear velocity readings**
+
+   We used three movement sensors to satisfy these requirements:
+
+   1. A [SparkFun GPS-RTK-SMA Breakout](https://www.sparkfun.com/products/16481) [movement sensor](/components/movement-sensor/) configured as a [`gps-nmea-rtk-serial`](/components/movement-sensor/gps/gps-nmea-rtk-serial/) model, providing GPS position measurements.
+   2. A [`wheeled-odometry`](/components/movement-sensor/wheeled-odometry/) model gathering angular and linear velocity information from the [encoders](/components/encoder/) wired to our base's [motors](/components/motor/).
+   3. A [`merged`](/components/movement-sensor/merged/) model aggregating the readings together for the navigation service to consume.
+
+{{% alert title="Tip" color="tip" %}}
+
+If you are using different hardware than us, the navigation setup process will be mostly the same, but you will need to substitute your robot's components.
+
+{{% /alert %}}
+
+Before you start, make sure to create a robot in [the Viam app](https://app.viam.com) and [install `viam-server`](/installation/) on your robot.
+
+## Configure the components you need
+
+First, configure the components of your robot.
+If you are using different _models_ of hardware, adjust your configuration accordingly.
+
+### Configure a board with `"digital_interrupts"`
+
+First, configure the [board](/components/board/) local to your rover.
+Follow [these instructions](/components/board/#configuration) to configure your board model.
+We used a [`jetson` board](/components/board/jetson/), but you can use whatever model of board you have on hand, as the [resource's API](/components/board/#api) is hardware agnostic.
```

Comment:
whatever is slightly too colloquial I think
```suggestion
We used a [`jetson` board](/components/board/jetson/), but you can use any model of board you have on hand, as the [resource's API](/components/board/#api) is hardware agnostic.
```

Reply:
Ok makes sense, done